subliminal/providers/legendastv.py

```diff
@@ -0,0 +1,466 @@
+# -*- coding: utf-8 -*-
+import json
+import logging
+import os
+import re
+
+from babelfish import Language, language_converters
+from datetime import datetime
+from guessit import guess_file_info
+from rarfile import RarFile, is_rarfile
+from requests import Session
+from tempfile import NamedTemporaryFile
+from zipfile import ZipFile, is_zipfile
+
+from . import ParserBeautifulSoup, Provider
+from ..cache import region, EPISODE_EXPIRATION_TIME, SHOW_EXPIRATION_TIME
+from ..exceptions import AuthenticationError, ConfigurationError
+from ..subtitle import Subtitle, fix_line_ending, guess_matches
+from ..video import Episode, Movie, SUBTITLE_EXTENSIONS
+
+TIMEOUT = 10
+
+logger = logging.getLogger(__name__)
+
+
+class LegendasTvSubtitle(Subtitle):
+    provider_name = 'legendastv'
+
+    def __init__(self, language, page_link, subtitle_id, name, imdb_id=None, guess=None, type=None, season=None,
+                 year=None, no_downloads=None, rating=None, featured=False, multiple_episodes=False, timestamp=None):
+        super(LegendasTvSubtitle, self).__init__(language, page_link=page_link)
+        self.subtitle_id = subtitle_id
+        self.name = name
+        self.imdb_id = imdb_id
+        self.type = type
+        self.season = season
+        self.year = year
+        self.no_downloads = no_downloads
+        self.rating = rating
+        self.featured = featured
+        self.multiple_episodes = multiple_episodes
+        self.timestamp = timestamp
+        self.guess = guess  # Do not need to guess it again if it was guessed before
+
+    @property
+    def id(self):
+        return '%s (%s)' % (self.name, self.subtitle_id)
```

Comment:
I can respond this one: yes.

eache download id(link) has one rar file and it contains multiple subs:

See this log:
https://gist.github.com/fernandog/7103aeea8c272395939f

If you don't add the name to it, all IDs would be 5660810a16e42, so that's why he added the name